root/report/RecordingsWithFutureDates.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+
+import RecordingRelationshipList from './components/RecordingRelationshipList';
+import type {ReportDataT, ReportRecordingRelationshipT} from './types';
+
+const RecordingsWithFutureDates = ({
+  $c,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportRecordingRelationshipT>): React.Element<typeof Layout> => (
+  <Layout
+    $c={$c}
+    fullWidth
+    title={l('Recordings with dates in the future')}
```

Comment:
The title suggests the recording itself has dates, maybe "Recordings with relationships having dates in the future" would be better? It's long, but so are other report titles.

Reply:
fixed